src/routes/auth/new-user/+page.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+// @ts-nocheck
+
+    import { page } from "$app/stores";
+    import welcome from '$lib/images/svelte-welcome.png'
```

Comment:
We can host our images here for now, but we'll have to move to a CDN at some point.